scripts/steps-init.yml

```diff
@@ -21,3 +21,7 @@ steps:
 - powershell: ./install-iqsharp.ps1
   displayName: "Installing IQ#"
   workingDirectory: $(System.DefaultWorkingDirectory)/scripts
+
+- task: NuGetAuthenticate@1
+  inputs:
+    nuGetServiceConnections: 'public-alpha feed'
```

Comment:
Since 2023.07.01 we need the copy of the released packages in the alpha feed. 
This change enables that.